CHANGELOG.md

```diff
@@ -1,13 +1,21 @@
-## [project-title] Changelog
+# FL Accelerator Changelog
 
-<a name="x.y.z"></a>
-# x.y.z (yyyy-mm-dd)
+##  September 2022 release
 
-*Features*
-* ...
+We are excited to announce the release of the September iteration of our [FL Accelerator repository](https://github.com/Azure-Samples/azure-ml-federated-learning).
 
-*Bug Fixes*
-* ...
+Here are the new features.
+- [Single location](https://github.com/Azure-Samples/azure-ml-federated-learning/blob/release-sdkv2-iteration-02/README.md) for all documentation.
+- Refactored repository - removed redundant directories/files.
+- Auto-provisioning tools now provision a setup with _secured_ internal silos (using Bicep templates), with extra VNet protection to prevent from exfiltrating data during an FL job. Connect your own storage accounts to this setup, and easily run FL jobs on your own data!
```

Comment:
proposed rewording:
- Auto-provisioning scripts to create a complete sandbox using bicep templates, by default an open sandbox on 3 regions, an experimental script with vnets and private endpoints is also available.
- A quickstart to run an FL demo in minutes, and easily run FL jobs on your own data!